tests/test_individual.py

```diff
@@ -1,68 +1,73 @@
 import pytest
-
 from comma.hypothesis import Hypothesis
 from comma.individual import Individual
 import numpy as np
 from pathlib import Path
 
 
 class TestIndividual:
-
     @pytest.fixture
     def dir_params(self):
         return Path("parameters/")
 
     @pytest.fixture
     def expected_actions(self):
-        return np.array([
-            'maintain_physical_distance',
-            'stay_at_home',
-            'exercise',
-            'socialise',
-            'seek_help',
-            'negative_coping'
-        ])
+        return np.array(
+            [
+                "maintain_physical_distance",
+                "stay_at_home",
+                "exercise",
+                "socialise",
+                "seek_help",
+                "negative_coping",
+            ]
+        )
 
     @pytest.fixture
     def expected_status(self):
-        return np.array(-18.080000000000002)
+        return np.array(-12.510000000000002)
```

Comment:
You could remove the trailing digits and use an [approximate comparison](https://docs.pytest.org/en/7.1.x/reference/reference.html#pytest-approx), which is safer anyway to prevent test failures due to slight changes in float point operations on different machines.
 ```suggestion
         return pytest.approx(np.array(-12.51))
```

Also, I am not sure why this is an array and not just a float.